src/js/views/globals/search/patient-search.scss

```diff
@@ -61,11 +100,40 @@
   }
 }
 
-.patient-search__no-results {
+.patient-search__info {
   color: $black-60;
   font-size: 14px;
   font-style: normal;
-  margin: 0 8px;
   padding: 16px 0;
   text-align: center;
 }
+
+.patient-search__tip {
+  border-bottom: 1px solid $black-90;
+  padding-bottom: 16px;
+  padding-left: 64px;
+  text-align: left;
+}
+
+.patient-search__search-by {
+  padding: 16px 0;
+  padding-left: 64px;
+  text-align: left;
+}
+
+.patient-search__search-by-title {
+  color: #999;
+  font-size: 12px;
+}
+
+.patient-search__search-by-buttons {
+  padding: 8px 0;
+}
+
+.patient-search__shortcut {
+  bottom: 0;
+  padding-bottom: 24px;
+  position: absolute;
+  text-align: center;
+  width: 100%;
+}
```

Comment:
Haven't had a chance to go through and clean this file up.

So this is still in a "get things working" state. Open to any suggestions here.